djmoney/contrib/exchange/backends/openexchangerates.py

```diff
@@ -18,4 +18,4 @@ def __init__(self, url=settings.OPEN_EXCHANGE_RATES_URL, access_key=settings.OPE
         self.access_key = access_key
 
     def get_params(self):
-        return {"app_id": self.access_key}
+        return {"app_id": self.access_key, "base": settings.BASE_CURRENCY}
```

Comment:
```suggestion
        return {"app_id": self.access_key, "base": getattr(settings, "BASE_CURRENCY", "USD")}
```

Reply:
It seems like current documentation doesn't require `BASE_CURRENCY` to be set but indicates that "USD" is the default.

> Backends return rates for a base currency, by default it is USD, but could be changed via BASE_CURRENCY setting. Open Exchanger Rates & Fixer supports some extra stuff, like historical data or restricting currencies in responses to the certain list. In order to use these features you could change default URLs for these backends:

Reply:
`update_rates` uses it as a default for a keyword argument. 
https://github.com/django-money/django-money/blob/c9749716fb9d98fe4bacb883798a2f28ee2c460a/djmoney/contrib/exchange/backends/base.py#L63

`BASE_CURRENCY` from djmoneny.settings does have a default, `BASE_CURRENCY` from django.conf.settings` doesn't so we are good here I guess.

Reply:
Ay! You're right, thanks! Not sure why I missed that.